source/numerics.tex

```diff
@@ -1203,28 +1214,22 @@
 \indextext{overloads!floating-point}%
 The additional overloads shall be sufficient to ensure:
 \begin{itemize}
-        \item If the argument has type \tcode{long double}, then it is effectively
-              cast to \tcode{complex<long double>}.
-        \item Otherwise, if the argument has type \tcode{double} or an integer type,
-              then it is effectively cast to \tcode{complex<\brk{}double>}.
-        \item Otherwise, if the argument has type \tcode{float}, then it is
-              effectively cast to \tcode{complex<float>}.
+\item
+If the argument has a floating-point type \tcode{T},
+then it is effectively cast to \tcode{complex<T>}.
+\item
+Otherwise, if the argument has integer type,
+then it is effectively cast to \tcode{complex<double>}.
```

Comment:
We should probably use `\keyword` for `double` etc., but we can fix that later.

Reply:
I don't think `\keyword` is used on the library clauses.

Reply:
At least not with any form of consistency. @jensmaurer Do you recall if we have a policy/plan for using `\keyword` in the library?

(No immediate action needed here.)

Reply:
> Do you recall if we have a policy/plan for using \keyword in the library?

We haven't discussed this in any detail, but if the plan for C++ is approximately that of C in that we want all mentions of keywords colored a little bit (or so), then we should use `\keyword` everywhere in prose text and configure `codeblock` accordingly.

Currently, we really want all keywords in the core section to be annotated, because there will be index entries for them.